kwok/charts/values.yaml

```diff
@@ -30,7 +30,7 @@ serviceMonitor:
   # -- Endpoint configuration for the ServiceMonitor.
   endpointConfig: {}
 # -- Number of replicas.
-replicas: 2
+replicas: 1
```

Comment:
Wonder if we should disable leader election when setting replicas to 1. Doesn't appear to be necessary, and if you've tested this without issues we don't need to do this. If we wanted to though, something along the lines of this should work:
```yaml
- name: LEADER_ELECT
  value: {{ int .Values.replicas | eq 1 | ternary "false" "true" | quote }}

```

Reply:
my gut feeling is we can just opt for simplicity here and keep it as is. Leader election usually runs into problems when there's more vying to be the leader, not less.